frontend/src/metabase/components/Tooltip.jsx

```diff
@@ -144,6 +117,22 @@ export default class Tooltip extends Component {
   };
 
   render() {
-    return React.Children.only(this.props.children);
+    const { isEnabled, tooltip } = this.props;
+    const isOpen =
+      this.props.isOpen != null ? this.props.isOpen : this.state.isOpen;
+    return (
+      <React.Fragment>
```

Comment:
🎊 I forgot about Fragment! Convenient here since I wouldn't be surprised if an additional div would have broken styling somewhere.